plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/RemoteDockerNode.java

```diff
@@ -51,6 +52,10 @@
     private final String                         nodeHost;
     private final String                         nodeIp;
 
+    private volatile boolean isBound;
```

Comment:
This is definitely flag. Can you elaborate on what are the pros in usage of enum here?